assets/src/blocks/Covers/CoversEditor.js

```diff
@@ -0,0 +1,153 @@
+import {
+  SelectControl,
+  PanelBody
+} from '@wordpress/components';
+import { useEffect } from '@wordpress/element';
+
+import { InspectorControls } from '@wordpress/block-editor';
+import TagSelector from '../../components/TagSelector/TagSelector';
+import PostSelector from '../../components/PostSelector/PostSelector';
+import PostTypeSelector from '../../components/PostTypeSelector/PostTypeSelector';
+import { Covers, COVER_TYPES } from './Covers';
+import { getCoversClassName } from './getCoversClassName';
+import { useCovers } from './useCovers';
+
+const { RichText } = wp.blockEditor;
+const { __ } = wp.i18n;
+
+const renderEdit = (attributes, toAttribute) => {
+  const { initialRowsLimit, posts, tags, cover_type, post_types } = attributes;
+
+  const rowLimitOptions = [
+    { label: __('1 Row', 'planet4-blocks-backend'), value: 1 },
+    { label: __('2 Rows', 'planet4-blocks-backend'), value: 2 },
+    { label: __('All rows', 'planet4-blocks-backend'), value: 0 },
```

Comment:
Using 0 for this is a bit in accurate :sweat_smile: If we make the value `null`, it unsets the attribute, which makes sense, we're removing the limit.

Reply:
If I change this to use `null` instead, there is a warning in the console when selecting that value:
```
Received NaN for the `value` attribute. If this is expected, cast the value to a string.
```
...and it's no longer possible to select "All rows"!

Reply:
That's because `onChange` is doing `Number(value)`, I'm not sure if we even need to do that, since the values are already numbers. But if we do need to cast them, we could make an exception for the "all rows" case.

Reply:
I've tested and you do need the cast since `SelectControl` converts the values to strings... And if we set the value to `null`, then it takes the label instead (`All rows`) for the value and leads to the warning in the console that I previously wrote. Now, I can remove the cast for this specific case and try to set `null` but in that case I get another console warning:
```
Warning: `value` prop on `select` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.
```

Reply:
Right we can use an empty string instead. But we don't need to set a value if we show all rows, so using 0 here is not really needed.

Reply:
This one is not a blocker if it turns out complicated to solve, just seemed defensive to not use 0 so that we can't inadvertently hide all rows. But maybe easier to consider separately afterwards :slightly_smiling_face: